README.rst

```diff
@@ -30,9 +30,9 @@ To see what coala can do for your language, run:
 
 .. Start ignoring LineLengthBear
 
-======================================= ================================================= ====================================================== =========================================================
-`Official Website <http://coala.io/>`__  `Twitter <https://twitter.com/coala_analyzer>`__ `Facebook <https://www.facebook.com/coalaAnalyzer/>`__ `Video Demo <https://asciinema.org/a/42968?autoplay=1>`__
-======================================= ================================================= ====================================================== =========================================================
+======================================= ===================================================== ================================================= ====================================================== =========================================================
```

Comment:
it appears that this change is unrelated to this commit :D i dont think it should be changed here